features/bootstrap/GraphqlContext.php

```diff
@@ -97,6 +98,48 @@ public function ISendTheGraphqlRequestWithOperation(string $operation)
         $this->sendGraphqlRequest();
     }
 
+    /**
+     * @When I have the following files for GraphQL request:
+     */
+    public function iHaveTheFollowingFilesForGraphqlRequest(TableNode $table)
+    {
+        $files = [];
+
+        foreach ($table->getHash() as $row) {
+            if (!isset($row['name']) || !isset($row['file'])) {
+                throw new \Exception("You must provide a 'name' and 'file' column in your table node.");
+            }
+
+            if (is_string($row['file']) && '@' == substr($row['file'], 0, 1)) {
```

Comment:
Why the `@`?

Reply:
You are right its not needed, because it was file based on habit I added it.I remove it.